modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -94,6 +93,54 @@ def create
 
         private
 
+        def validate_decide_params!(proc_id:, decision:)
+          if proc_id.blank?
+            raise ::Common::Exceptions::ParameterMissing.new('procId',
+                                                             detail: 'procId is required')
+          end
+
+          unless decision.present? && %w[accepted declined].include?(decision)
+            raise ::Common::Exceptions::ParameterMissing.new(
+              'decision',
+              detail: 'decision is required and must be either "ACCEPTED" or "DECLINED"'
+            )
+          end
+        end
+
+        def send_declined_notification(ptcpnt_id:, first_name:, representative_id:)
+          lockbox = Lockbox.new(key: Settings.lockbox.master_key)
+          encrypted_ptcpnt_id = lockbox.encrypt(ptcpnt_id)
+          encrypted_first_name = lockbox.encrypt(first_name)
+
+          ClaimsApi::VANotifyDeclinedJob.perform_async(encrypted_ptcpnt_id:, encrypted_first_name:,
```

Comment:
When this call happens, I get a 500 with the error below
```
#<Encoding::UndefinedConversionError: "\x8D" from ASCII-8BIT to UTF-8>
```

Reply:
The encryption appears to switch those over to that `ASCII-8BIT` which it _looks_ like it doesn't like (this is a first seeing this for me)

<img width="794" alt="Screenshot 2024-11-15 at 9 44 46 AM" src="https://github.com/user-attachments/assets/833a4633-0ae6-4dbe-b731-9bef01d5525b">

Reply:
In case it helps, these are the values I am seeing going into the encryption:

- ptcpnt_id: `"600049322"`
- first_name: `ANDREA`

Reply:
Any chance this error was from the `procId` bug (now [fixed](https://github.com/department-of-veterans-affairs/vets-api/pull/19362/commits/1800d14ffa462b1adbbb703259e9193a108b90f5))? I was able to get an email end-to-end testing:

<img width="814" alt="200" src="https://github.com/user-attachments/assets/5bdd1044-f83d-4319-9709-ae13fecbfd22">
<img width="1007" alt="email" src="https://github.com/user-attachments/assets/b9111d7d-4ed8-47e0-bb03-e7d008de7cfa">

Reply:
Oh never mind, I was using `.new.perform` in my testing. `perform_async` does raise an `ASCII-8BIT to UTF-8` error (investigating).

Reply:
Ya I had made that procID update locally to get to that point.  Interesting that new.perform does not raise that, I am going to have to remember that part 'cause that is going to catch me at some point I bet

Reply:
Ok, [encoded](https://github.com/department-of-veterans-affairs/vets-api/pull/19362/commits/4953a3483a08dba8aedbd9110f9b6808f49dd8d3) the encrypted params with base64 🧅😅. We should be ok now. Good catch!

<img width="1212" alt="encoded" src="https://github.com/user-attachments/assets/6a172e26-6160-453b-a60f-b800b6e2978c">

Reply:
It also queues up for me, curious if that did not error out for you right after that though? 

For me that still errors out on me though when trying to follow this from the beginning to the end. 

```
ArgumentError: wrong number of arguments (given 1, expected 0; required keywords: encrypted_ptcpnt_id, encrypted_first_name, representative_id)
```

Reply:
What is `representative_id_val`? Is that a local change you made?

Reply:
oh sorry , ya let me get a better screenshot,  I was trying to find out why the params where giving me problems, one sec

Reply:
@tycol7 Here you go, apologies for the wrong image there
![Screenshot 2024-11-15 at 2 09 31 PM](https://github.com/user-attachments/assets/9541b822-307a-4bd5-983d-dc3595192c13)

![Screenshot 2024-11-15 at 2 09 56 PM](https://github.com/user-attachments/assets/d91c1964-81d0-4ce1-8c09-d6019ecffeff)

Reply:
Turns out sidekiq does not like keyword arguments, so [changing to ordered arguments](https://github.com/department-of-veterans-affairs/vets-api/pull/19362/commits/bc6ef6b47f6e9749c7d37bcd0894bdcd4a1998b4) (seems to do the trick).